packages/redux-query/index.d.ts

```diff
@@ -25,13 +25,13 @@ declare module 'redux-query' {
   }
   export type EntitiesState = Entities;
 
-  export enum HttpMethods {
-    DELETE,
-    GET,
-    HEAD,
-    POST,
-    PUT,
-    PATCH,
+  export const enum httpMethods {
```

Comment:
Can we keep the H uppercase? to be consistent with constant variable types naming conventions

Reply:
We can't, as JS part exports with lowercase 'h' and TS has to follow that to be usable.

Reply:
ah ok. Sounds good!